tooling/src/hypothesistooling/__init__.py

```diff
@@ -196,6 +195,8 @@ def changed_files_from_master():
 DEPLOY_KEY = os.path.join(SECRETS, 'deploy_key')
 PYPIRC = os.path.join(SECRETS, '.pypirc')
 
+RUBYGEMS_API_KEY = os.path.join(SECRETS, 'api_key.yaml')
```

Comment:
minor: why is this defined in the top-level tooling file, if it’s a Ruby-specific constant?

Reply:
It's useful for the secret decryption code to know what files it's expecting so that it can check it's decrypted successfully and fail there (we have a bunch of asserts about that). Although, I think I didn't add an assert for it and should...